src/components/SavedProvidersList/saved-providers-list.css

```diff
@@ -68,3 +68,14 @@
 .print-email-btn:focus {
   outline: none;
 }
+
+.email-icon, .print-icon {
+  color: #8c45cf;
+}
+.email-icon:hover, .print-icon:hover {
+  cursor: pointer;
```

Comment:
nit: you can move cursor: pointer into the .email-icon, .print-icon rule. The browser will only change the cursor to a pointer when the mouse is over the icon, so no need for :hover